src/app/components/DetailsPage/ConditionsTable.tsx

```diff
@@ -1,14 +1,14 @@
 import { Bullseye } from '@patternfly/react-core';
 import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
-import { Device } from '@types';
+import { Condition } from '@types';
 import * as React from 'react';
 
 type ConditionsTableProps = {
-  device: Device;
+  conditions: Array<Condition> | undefined;
 };
 
-const ConditionsTable: React.FC<ConditionsTableProps> = ({ device }) => {
-  return device.status?.conditions?.length ? (
+const ConditionsTable: React.FC<ConditionsTableProps> = ({ conditions }) => {
+  return conditions?.length ? (
     <Table aria-label="Device conditions table">
```

Comment:
We could probably add a "type" prop and use it for the `aria-label`, and also for the empty state text.

```suggestion
    <Table aria-label={`${type} conditions table`}>
```

Reply:
fixed